scripts/musicbrainz-initial-dump/importArtists.js

```diff
@@ -0,0 +1,77 @@
+//NEXT STEPS:
+//2. be sure duplicates are not saved
+//5. test in lambda
+//6. execute locally
+
+const db = require('../../models');
+const { Client }  = require('pg');
+const mongoose = require('mongoose');
+
+const importSource = "MusicBrainz";
+
+mongoose.connect(process.env.MONGO_CONNECTION);
+const pgClient = new Client()
+
+exports.myHandler = async function(event, context, callback) {   
+	console.log('Starting MusicBrainz import process.');
+	try {
+		//Connect to MusicBrainz DB and run a query for all artists
+		//Import MusicBrainz artists into a Mongo database
+		
+		var importProgress = await db.ImportProgress.findOne();
+		if (importProgress == null) {
+			importProgress = new db.ImportProgress({artist_import: { last_imported_id: 0 }});
+		}
+		
+		var recordsToProcess = 50;
```

Comment:
This is 50 now, but I'll probably make it something like 500 or 1000 when running the actual import.